crown-court-contribution/src/main/resources/schemas/CreateContributionRequest.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "CreateContributionRequest.json",
+  "type": "object",
+  "title": "Create Contribution Request",
+  "description": "Data contract for creating contribution records via the Court Data API",
+  "properties": {
+    "repId": {
+      "type": "integer",
+      "description": "The associated MAAT/REP ID"
+    },
+    "applId": {
+      "type": "integer",
+      "description": "The associated application ID"
+    },
+    "userCreated": {
+      "type": "string",
+      "description": "The user creating the record"
+    }
+  },
```

Comment:
Missing laaTransactionId.

Reply:
This field is being passed as a header with all requests, which I think should be our preferred method. However, there's probably some work to do in the Court Data API to make sure we're consuming this and logging the value correctly.